qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -95,19 +95,24 @@ def wait_for_logstash(service)
       run_logstash_instance(input_config_name, all_config_options) do |downstream_logstash_service|
         run_logstash_instance(output_config_name, all_config_options) do |upstream_logstash_service|
 
+          file_output_path = File.join(downstream_logstash_service.logstash_home, output_file_path_with_datetime)
+
           try(num_retries) do
             downstream_event_stats = downstream_logstash_service.monitoring_api.event_stats
+            upstream_event_stats = upstream_logstash_service.monitoring_api.event_stats
```

Comment:
The dangerous part here, upstream has a generator input, which we don't have guarantee it and its monitoring API are live. That's why we didn't include the upstream expectation.